docs/resources/license_policy.md

```diff
@@ -0,0 +1,150 @@
+---
```

Comment:
Per https://github.com/hashicorp/terraform-plugin-docs#conventional-paths, the path for this file should be `templates/resources/license_policy.md.tmpl`, etc.

Reply:
tfplugindocs can't parse resources schema when we use a template (we've tried with @alexhung). I'm keeping it the old way when docs for the resources are generated from `examples/resources` without the template usage

Reply:
This implies something isn't correct/valid with the license policy schema and the plugin chokes on that. Keep this is ok for now but we need to resolve this issue at some (not so distant) future. We can't kick this can down the road.